core/src/apps/management/recovery_device/homescreen.py

```diff
@@ -48,33 +46,77 @@
     in_progress = storage.recovery.is_in_progress()
     word_count = storage.recovery.get_word_count()
     dry_run = storage.recovery.is_dry_run()
+    backup_type = storage.recovery.get_backup_type()
```

Comment:
`word_count` is being used few lines above:

```
    if not word_count:  # the first run, prompt word count from the user
        word_count = await _request_and_store_word_count(ctx, dry_run)
```

I didn't want to inline it, because the function was quite long before. That being said, it might be more readable now. I have moved the `backup_type` for now and left the rest, but if you want, feel free to inline the code.

Reply:
i played around with the code a little and decided to inline it, it makes more sense than keeping it separate

Reply:
I've looked at it shortly and it looks good to me! I lost perspective after I've move it 10 times back and forth :)